engine/table/src/main/java/io/deephaven/engine/table/impl/perf/QueryPerformanceNugget.java

```diff
@@ -221,43 +243,44 @@ private QueryPerformanceNugget() {
         description = null;
         isUser = false;
         inputSize = NULL_LONG;
+        onCloseCallback = null;
 
         authContext = null;
         callerLine = null;
 
-        startClockEpochNanos = NULL_LONG;
-
         state = null; // This turns close into a no-op.
         shouldLogThisAndStackParents = false;
     }
 
-    public void done() {
-        done(QueryPerformanceRecorder.getInstance());
+    public void markStartTime() {
```

Comment:
`start()`? Collapse with base entry start? Document as doing a base entry start? Should probably be synchronized.